### PR TITLE
adjusted routes/models

### PR DESCRIPTION
**Pull request by roshan-rm01**
I created a template for the job operations (create a job, retrieve a job, update job, delete job) though I was unsure how I can save it to the Mongo database. So far the 'create' route should be working well though the rest of the routes need adjusting because it has to query the MongoDB to get and adjust the results. Moreover, I added an id field for the applicant, organisation and job models.

### CODE REVIEW

File: backend/models/applicant/model.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Education(EmbeddedModel):
     
 
 class Applicant(Model):
+    id: int
```
Comment (Youngestdev) [inline backend/models/applicant/model.py]: Oh, there's no need for this. IDs are generated automatically.

File: backend/models/organisation/model.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 
 class Organisation(Model):
+    id: int
```
Comment (Youngestdev) [inline backend/models/organisation/model.py]: Same comment as above.

File: backend/models/organisation/model.py [hunk 3]
```diff
@@ -16,6 +17,7 @@ class Organisation(Model):
     companySize: Optional[str] = None
     
 class Job(Model):
+    id: int
```
Comment (Youngestdev) [inline backend/models/organisation/model.py]: Same comment as above.

File: backend/routes/organisation/route.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+@app.get('/job/get/{job_id}/', response_model=Job)
+async def get_job(job_id: int):
+    return int
+
+@app.put('/job/update/{job_id}/', response_model=Job)
+async def update_job(job: Job):
+    return job
+
+@app.delete('job/delete/{job_id}/'):
+    return {'model': 'deleted'}
```
Comment (Youngestdev) [inline backend/routes/organisation/route.py]: I will send a PR of how I wrote the applicant's functionalities. The code there will give you an insight on how to write this too.

**Review by Youngestdev — CHANGES_REQUESTED** [review]

File: backend/routes/organisation/route.py (hunk 4 above)
Comment (roshan-rm01) [reply backend/routes/organisation/route.py]: Adjusted routes and got rid of id field

File: backend/models/organisation/model.py [hunk 5]
```diff
@@ -1,28 +1,75 @@
...
+                    "firstName": "Larry",
+                    "lastName": "Page",
+                    "email": "page@gmail.com",
+                    "password": "Strong+Password",
+                    "companyName": "Google",
+                    "location": "USA",
+                    "companyBio": "Google is the largest technology company",
+                    "industry": "Technology",
+                    "telephone": "+447398346125",
+                    "companySize": "10,000+ Employees"
```
Comment (Youngestdev) [inline backend/models/organisation/model.py]: It will be nice to limit the data the user can see about the organisation so I'll fix this.

**Review by Youngestdev — COMMENTED** [review]
There are a few changes to be made. I'll check through and fix then.

**Pull request merged**